src/Firebase/Auth/ApiClient.php

```diff
@@ -191,22 +191,25 @@ public function verifyPassword(string $email, string $password): ResponseInterfa
 
     /**
      * @param string $idToken
+     * @param string $continueUrl
      *
      * @return ResponseInterface
      */
-    public function sendEmailVerification(string $idToken): ResponseInterface
+    public function sendEmailVerification(string $idToken, string $continueUrl = null): ResponseInterface
     {
         return $this->request('getOobConfirmationCode', [
             'requestType' => 'VERIFY_EMAIL',
             'idToken' => $idToken,
+            'continueUrl' => $continueUrl,
         ]);
     }
 
-    public function sendPasswordResetEmail(string $email): ResponseInterface
+    public function sendPasswordResetEmail(string $email, string $continueUrl = null): ResponseInterface
     {
         return $this->request('getOobConfirmationCode', [
             'email' => $email,
             'requestType' => 'PASSWORD_RESET',
+            'continueUrl' => $continueUrl,
```

Comment:
Please wrap the array in `array_filter(...)` to avoid sending an empty `continueUrl`field if none is set (the small things :D)